metricbeat/docker-compose.yml

```diff
@@ -51,6 +52,8 @@ services:
       jolokia:        { condition: service_healthy }
       kafka:          { condition: service_healthy }
       kibana:         { condition: service_healthy }
+      kubernetes:     { condition: service_healthy }
+      kube-state:     { condition: service_started }
```

Comment:
kube-state-metrics doesn't provide health service, but as soon as it starts it should work so we can live with this it seems